tests/validation/tests/v1_api/test_monitoring_v2.py

```diff
@@ -0,0 +1,125 @@
+from .common import *  # NOQA
+import pytest
+import requests
+import semver
+
+namespace = {'client': None, 'cluster': None, 'rancher_catalog': None, 'project': None}
+monitoring_v2 = 'rancher-monitoring'
+monitoring_v2_version = os.environ.get('RANCHER_MONITORING_V2_VERSION', None)
+monitoring_v2_app_id = 'cattle-monitoring-system/rancher-monitoring'
+monitoring_v2_crd_app_id = 'cattle-monitoring-system/rancher-monitoring-crd'
+monitoring_v2_namespace = "cattle-monitoring-system"
+
+
+def test_install_monitoring_v2():
+    install_monitoring()
+
+
+def install_monitoring():
+    client = namespace['client']
+    rancher_catalog = namespace['rancher_catalog']
+    # install the monitoring v2 app into the new project
+    project_id = namespace["project"]["id"]
+    cluster_id = namespace["cluster"]["id"]
+    cluster_name = namespace["cluster"]["spec"]["displayName"]
+    values = read_json_from_resource_dir("monitoring_v2_values.json")
+    values["projectId"] = project_id
+    for chart in values["charts"]:
+        chart["version"] = monitoring_v2_version
+        chart["projectId"] = project_id
+        chart["values"]["global"]["cattle"]["clusterId"] = cluster_id
+        chart["values"]["global"]["cattle"]["clusterName"] = cluster_name
+    res = client.action(rancher_catalog, "install", values)
+    # wait 2 minutes for the app to be fully deployed
+    time.sleep(120)
+    # check the app rancher-monitoring-crd first, then the app rancher-monitoring
+    wait_for(lambda: client.by_id_catalog_cattle_io_app(monitoring_v2_crd_app_id).status.summary.state == "deployed",
```

Comment:
Please run flake flake8 against all files.

Reply:
I have fixed most of the warnings from the flake8 checks.